content/en/blog/tracing-syscalls-using-ebpf/index.md

```diff
@@ -0,0 +1,278 @@
+---
+title: Tracing System Calls Using eBPF -Vol. 1
+linktitle: Tracing System Calls Using eBPF -Vol. 1
+description: Learn how to trace system calls using eBPF programms.
+date: 2023-09-05
+author: Anshu Bansal, Rakshit Awasthi, Ashutosh Venkatrao More
+slug: tracing-syscalls-using-ebpf
+images:
+  - /blog/tracing-syscalls-using-ebpf/images/ebpf-blog-featured.png
+tags: ["eBPF", "Falco", "Syscalls"] 
+---
+
+## Introduction:
```

Comment:
Can you specify it will be a 2 parts blog post.

Reply:
done